### PR TITLE
Create styles

### PR DESCRIPTION
**Pull request by ejdelsztejn**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Testing - building tests
- [ ] Formatting - no new features
- [ ] Styling- no new features

### Detailed Description (Why is this change required? What problem does it solve? Any challenges that arose while implementing this feature? If so, how were the addressed?)
This branch contains:
- The migration for the `styles` table
- The `Style` model
- The one-to-many relationship between `Style` and `Products`

##### Questions
@Chulstro my understanding was that we would create the many-to-many relationship between `Users` and `Styles` through a `UserStyles` table, but I just wanted to clarify.  That association can be made in the models once the `UserStyles` table exists.

### CODE REVIEW

**Review by amykarnaze — APPROVED** [review]
This is a fun first peak into working with the BE! Nice tables 👍

File: code/api/src/config/database.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "development": {
-    "database": "crate_development",
+    "database": "crate",
```
Comment (Chulstro) [inline code/api/src/config/database.json:3]: Did we create a new database?

File: code/api/src/modules/style/model.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+module.exports = function(sequelize, DataTypes) {
+  let Style = sequelize.define('styles', {
+    name: {
+      type: DataTypes.STRING
+    }
+  })
+
+  Style.associate = function(models) {
+    // Style.hasMany(models.User) --> aren't we using UserStyle for this?
```
Comment (Chulstro) [inline code/api/src/modules/style/model.js:11]: UserStyle will belongto style and user. Theoretically should look like subscriptions relationships

**Comment by Chulstro** [comment]
So I think the only thing that still needs to happen is that we need to create a new column for product in order for it to belong to a style

**Review by Chulstro — APPROVED** [review]

**Review by nickhartdev — APPROVED** [review]
Really cool seeing all of this come together! Thanks for doing this

**Review by AurumValian — COMMENTED** [review]
This is really cool! Good job condensing those 359k lines 😂

**Pull request merged**